### DISCUSSION TITLE
WeeWX Cumulative User Definable

### DISCUSSION THREAD

**Discussion by gary-hammer** [Daveiano/weewx-wdc#180] [Ideas]
I think this makes more sense than the in-built way for plots and reporting.
Basically user configurable reset times.

https://github.com/gjr80/weewx-xcumulative

https://github.com/gjr80/weewx-xcumulative/wiki/Available-config-options

https://github.com/gjr80/weewx-xcumulative/wiki/Plot-use

**Comment by Daveiano** [comment]
Interesting! I just tried this out and noticed the skin needs some tweaks to fully support the extension (and other custom XTypes in general). I will have a look into this and probably include the changes in the next (coming soon) next release!

**Comment by Daveiano** [comment]
Hi gary, you should now be able to use [weewx-xcumulative](https://github.com/gjr80/weewx-xcumulative) in the skin with the [latest release](https://github.com/Daveiano/weewx-wdc/releases/tag/v3.3.0)!

There are some example configs provided here: https://github.com/Daveiano/weewx-wdc/wiki/Code-Snippets#using-weewx-xcumulative

**Reply by gary-hammer** [reply]
Thanks.
That was quick work!
